pkg/target/vapp_dto.go

```diff
@@ -13,12 +13,22 @@ func (vapp *VirtualApp) BuildDTO() (*proto.EntityDTO, error) {
 		NewEntityDTOBuilder(proto.EntityDTO_VIRTUAL_APPLICATION, vapp.UUID).
 		DisplayName(vapp.Name)
 
-	if err := vapp.getCommoditiesBought(vAppBuilder); err != nil {
+	err, qpsused, qpscap, rtused, rtcap := vapp.getCommoditiesBought(vAppBuilder)
```

Comment:
why not to use the `Resource` structure,  the number of returned values can be reduced from 5 to 3?